src/components/CommentDetail/CommentDetail.tsx

```diff
@@ -177,7 +177,10 @@ export const CommentDetail = forwardRef<HTMLDivElement, CommentDetailProps>((pro
     onSendTip(comment);
     const contentType = 'comment';
     const referenceId = comment.id;
-    dispatch(setTippedContent(contentType, referenceId));
+
+    const isTippingForDotSamaTokensDisabled = false;
```

Comment:
can we generalized variable, from `isTippingForDotSamaTokensDisabled` to `isTippingDisabled`

Reply:
I'm afraid that it'll be confusing to change it into `isTippingDisabled` since it's not actually disabling tipping as a whole, but only for DotSama tokens, thus the long-name variable. We do have `isTippingDisabled` already so I don't want the two get mixed. @irmannmal

Reply:
or we can change it into `isDotSamaTokensDisabled` ? @irmannmal

Reply:
no, we need another general variable, this not for DOT or KSM, but for all coin/token @ical10